wrappers/python/openmm/app/xtcfile.py

```diff
@@ -65,7 +65,8 @@ def __init__(self, fileName, topology, dt, firstStep=0, interval=1, append=False
         else:
             if os.path.isfile(self._filename) and os.path.getsize(self._filename) > 0:
                 raise FileExistsError(f"The file '{self._filename}' already exists.")
-
+            #XTC requires the file to exist before writing
+            open(self._filename, "a").close()
```

Comment:
This isn't necessary.  `fopen()` with append mode already creates the file if it doesn't exist.